cardinal/world.go

```diff
@@ -99,13 +99,13 @@ func NewMockWorld(opts ...WorldOption) (*World, error) {
 // CreateMany creates multiple entities in the world, and returns the slice of ids for the newly created
 // entities. At least 1 component must be provided.
 func (w *World) CreateMany(num int, components ...component_metadata.Component) ([]EntityID, error) {
```

Comment:
Can/should we just get rid of these duplicates?

Reply:
As discussed the other Create and CreateMany in components is now being moved to components in the ecs folder.